input-explainer.md

```diff
@@ -336,29 +336,29 @@ The UA may update the `gamepad` state at any point, but it must remain constant
 
 ### XR gamepad mapping
 
-The WebXR Device API also introduces a new standard controller layout indicated by the `mapping` value of `xr-standard`. (Additional mapping variants may be added in the future if necessary.) This defines a specific layout for the inputs most commonly found on XR controller devices today. The following table describes the buttons/axes and their physical locations:
+The WebXR Device API also introduces a new standard controller layout indicated by the `mapping` value of `xr-standard`. (Additional mapping variants may be added in the future if necessary.) This defines a specific layout for the inputs most commonly found on XR controller devices today. The following table describes the buttons/axes and their associated physical inputs:
 
-| Button     | `xr-standard` Location            |
-| ---------- | --------------------------------- |
-| buttons[0] | Primary trigger                   |
-| buttons[1] | Primary Touchpad/Joystick click   |
-| buttons[2] | Grip/Secondary trigger            |
-| buttons[3] | Secondary Touchpad/Joystick click |
+| Button     | `xr-standard` Binding  |
+| ---------- | -----------------------|
+| buttons[0] | Primary trigger        |
+| buttons[1] | Grip/Secondary trigger |
+| buttons[2] | Touchpad press         |
+| buttons[3] | Thumbstick press       |
 
-| Axis    | `xr-standard` Location        |
-| ------- | ----------------------------- |
-| axes[0] | Primary Touchpad/Joystick X   |
-| axes[1] | Primary Touchpad/Joystick Y   |
-| axes[2] | Secondary Touchpad/Joystick X |
-| axes[3] | Secondary Touchpad/Joystick Y |
+| Axis    | `xr-standard` Binding |
+| ------- | ----------------------|
+| axes[0] | Touchpad X            |
+| axes[1] | Touchpad Y            |
+| axes[2] | Thumbstick X          |
+| axes[3] | Thumbstick Y          |
 
-Additional device-specific inputs may be exposed after these reserved indices, but devices that lack one of the canonical inputs must still preserve their place in the array. If a device has both a touchpad and a joystick the UA should designate one of them to be the primary axis-based input and expose the other at axes[2] and axes[3] with an associated button at button[3].
+Additional device-specific inputs may be exposed after these reserved indices, but devices that lack one of the canonical inputs must still preserve their place in the array.
 
 In order to make use of the `xr-standard` mapping, a device must meet **at least** the following criteria:
 
  - Is a `tracked-pointer` device. 
  - Has a trigger or similarly accessed button
```

Comment:
It seems fine to require at least a primary button/trigger for `buttons[0]`.

Should we more strongly state then that this button MUST also trigger `onselect` for this `XRInputSource`?